golang/.gitignore

```diff
@@ -1,3 +1,6 @@
 /.GOPATH
 /bin
 sources.go
+language-examples
+/scratch
+/outputtests
```

Comment:
What are these last 2?  The first, er, yeah, I kept getting that file so wise to include. :smile:

Reply:
I was just testing something and wasn't trying to push any of that. Will remove.